bentoml/yatai/yatai_service_impl.py

```diff
@@ -356,23 +356,45 @@ def UpdateBento(self, request, context=None):
     def DangerouslyDeleteBento(self, request, context=None):
         try:
             # TODO: validate request
-            bento_pb = self.bento_metadata_store.get(
-                request.bento_name, request.bento_version
-            )
-            if not bento_pb:
-                msg = (
-                    f"BentoService {request.bento_name}:{request.bento_version} "
-                    f"has already been deleted"
-                )
-                return DangerouslyDeleteBentoResponse(status=Status.ABORTED(msg))
+            bento_pb_list_for_delete = []
+            if request.all:
+                # Delete all bentos
+                bento_pb_list_for_delete = self.bento_metadata_store.list()
```

Comment:
pass this into `dangerously_batch_delete` instead of making `None` the default parameter?